packages/react-components/react-breadcrumb/library/README.md

```diff
@@ -2,8 +2,6 @@
 
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33385&commitId=d7ab75777e4d22c07628ffb7928cbe51580482c1&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|33| Changed|

</details>


  <details><summary><b>Drawer</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|2232| Changed|
|Drawer.Full Overlay Dark Mode.chromium.png|988| Changed|
|Drawer.Full Overlay RTL.chromium.png|1171| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>